bin/load-mocks

```diff
@@ -668,7 +668,7 @@ def main(num_events=1, extra_events=False):
                 create_alert_rule_trigger(alert_rule, "critical", AlertRuleThresholdType.ABOVE, 10)
                 create_incident(
                     org,
-                    type=IncidentType.DETECTED,
+                    _type=IncidentType.DETECTED,
```

Comment:
I usually think of `_` prefixed vars as unused or 'hidden'. Would a name like `incident_type` work here?

Reply:
Turn out pep has a recommendation for this:
`single_trailing_underscore_: used by convention to avoid conflicts with Python keyword, e.g.`
I'll use that instead